### PR TITLE
feat: migrate contributor_id in contribution table to be 0x prefixed

### PR DESCRIPTION
**Pull request by tdelabro**

### CODE REVIEW

File: migrations/2022-08-22-135718_move-contributorid-to-0x-prefixed-format/up.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+UPDATE contributions
+    SET contributor_id = '0x' || contributor_id
```
Comment (ofux) [inline migrations/2022-08-22-135718_move-contributorid-to-0x-prefixed-format/up.sql:2]:
```suggestion
    SET contributor_id = '0x' || contributor_id
    WHERE contributor_id LIKE '000%'
```

File: migrations/2022-08-22-135718_move-contributorid-to-0x-prefixed-format/down.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+UPDATE contributions
+    SET contributor_id = right(contributor_id, -2)
```
Comment (ofux) [inline migrations/2022-08-22-135718_move-contributorid-to-0x-prefixed-format/down.sql:2]:
```suggestion
    SET contributor_id = right(contributor_id, -2)
    WHERE contributor_id LIKE '0x%'
```

**Review by ofux — CHANGES_REQUESTED** [review]
I prefer to be as safe as possible when writing this kind of migrations ;)

File: migrations/2022-08-22-135718_move-contributorid-to-0x-prefixed-format/up.sql (hunk 1 above)
Comment (tdelabro) [reply migrations/2022-08-22-135718_move-contributorid-to-0x-prefixed-format/up.sql:2]: what about `WHERE contributor_id NOT LIKE '0x%'` instead

Comment (ofux) [reply migrations/2022-08-22-135718_move-contributorid-to-0x-prefixed-format/up.sql:2]: yes, sounds good

**Review by ofux — APPROVED** [review]

**Pull request merged**